docs/deprecations.md

```diff
@@ -6,6 +6,12 @@ layout: Doc
 
 # Serverless Framework Deprecations
 
+<a name="SERVICE_OBJECT_NOTATION"><div>&nbsp;</div></a>
+
+## `service.awsKmsKeyArn` property will no longer be used
```

Comment:
This title doesn't reflect deprecation we configured

Reply:
Sorry about that. I wasn't really sure if I can directly use only `service`. Can you suggest a better description for it ?

Reply:
You copied title from other deprecation, while title should reflect the deprecation it documents

What do you think will make a good title?

Reply:
I think it can be
## `service.awsKmsKeyArn` will be deprecated in next major release.

Reply:
Let's maybe put it as:

```
`service` property object notation
```

(it fells better to me to just mark context, instead of signalling deprecation - although I know some of the titles do not fit that)

Reply:
Got it @medikoo. I believe this is how I'll learn 👍 Thanks! I have pushed the updated heading.